ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/metrics/AuditMetrics.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.metrics;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+import com.codahale.metrics.Timer;
+import org.apache.cassandra.metrics.CassandraMetricsRegistry;
+
+/**
+ * Helper class to create and update audit metrics.
+ */
+public class AuditMetrics
+{
+    private static final String GROUP_NAME = "com.ericsson.bss.cassandra.ecaudit";
+    private static final String METRIC_TYPE = "Audit";
+    private static final String METRIC_SCOPE = "ClientRequests";
+    private static final String METRIC_NAME_FILTER = "AuditFilter";
+    private static final String METRIC_NAME_AUDIT = "Audit";
```

Comment:
How about calling things AuditFilter and AuditLogger? That way we keep the namespace open for a metric covering all parts if we feel that is needed in the future.

Reply:
Sure, I've updated it.